src/ReactWrapper.jsx

```diff
@@ -224,13 +225,14 @@ export default class ReactWrapper {
    * NOTE: can only be called on a wrapper instance that is also the root instance.
    *
    * @param {Object} state to merge
+   * @param {Function} cb - callback function
    * @returns {ReactWrapper}
    */
-  setState(state) {
+  setState(state, cb) {
     if (this.root !== this) {
       throw new Error('ReactWrapper::setState() can only be called on the root');
     }
-    this.instance().setState(state);
+    this.instance().setState(state, cb);
```

Comment:
Yea seems like it's taken care of here: 
https://github.com/facebook/react/blob/master/src/renderers/shared/stack/reconciler/ReactUpdateQueue.js#L257